libs/blocks/table/table.js

```diff
@@ -97,6 +97,36 @@ function handleHeading(table, headingCols) {
   });
 }
 
+function handleAddOnContent(table) {
+  const addOnKey = 'ADDON';
+  const addOns = [...table.querySelectorAll('.section-row-title')]
+    .filter((row) => row.innerText.includes(addOnKey));
+  if (addOns.length) {
+    table.classList.add('has-addon');
+    addOns.forEach((addOn) => {
+      const addOnRow = addOn.parentElement;
+      addOnRow.remove();
+      const [position, order, style] = addOn.innerText.split('_')
+        .filter((key) => key !== addOnKey).map((key) => key.toLowerCase());
+      if (position && order) {
+        [...table.querySelector('.row-heading').children].forEach((headCol) => {
+          const indexAttr = 'data-col-index';
```

Comment:
Nit: we can move this outside of the loop, since it doesn't get updated

Reply:
Done